test/auth-test.ts

```diff
@@ -0,0 +1,65 @@
+import { expect } from 'chai';
+import prismaInstance from '../src/db';
+import { endpoint } from './utils/config-test';
+import jwt from 'jsonwebtoken';
+
+const inputLogin = {
+  email: 'gabi@gmail.com',
+  password: 'coxinha123',
+  rememberMe: true,
+};
+
+const payloadId = 'df59e650-f751-4c1c-a991-61b9b798a5f9';
+
+describe('Auth user endpoint', () => {
+  it('Response and auth success', async () => {
+    const response = await endpoint.post('/auth', inputLogin);
+    const userLogged = (await prismaInstance.user.findUnique({
+      where: {
+        email: inputLogin.email,
+      },
+    }))!;
+    expect(response.status).to.be.deep.eq(200);
+    expect(response.data.user).to.be.deep.eq({
+      id: userLogged.id,
+      name: userLogged.name,
+      email: userLogged.email,
+      birthDate: userLogged.birthDate,
+    });
+    expect(response.data).to.have.property('token');
+    const payload = jwt.verify(response.data.token.split(' ')[1], String(process.env.SECRET_JWT));
+    expect(payload.id).to.be.deep.eq(payloadId);
```

Comment:
os testes deveriam estar seguindo o padrão [AAA](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80)